scripts/payload_examples/offboarding.yaml

```diff
@@ -1,7 +1,5 @@
 cluster:
-  - Smaug
-environment:
-  - moc
+  - moc/Smaug
```

Comment:
why don't we lowercase it as `moc/smaug`? Or write it as `MOC/Smaug`. `moc/Smaug` looks like a weird hybrid to me :laughing:

Reply:
we can go with the first one :+1:. I have tried to keep lower and upper case as it was before that, but this will be better :).